test/mock.utils.ts

```diff
@@ -24,21 +24,36 @@ export const loadMockSession = (mockLoadSessionFunction: jest.Mock<typeof loadSe
 };
 
 export const loadCompanyAuthenticatedSession = (mockLoadSessionFunction: jest.Mock<typeof loadSession>,
-                                                companyNumber?: string): void => {
+                                                companyNumber, email): void => {
```

Comment:
you could make email optional with email?, then you wouldn't need to pass "" as email in some tests.